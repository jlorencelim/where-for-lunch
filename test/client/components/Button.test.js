import React from 'react';
import renderer from 'react-test-renderer';
import Button from 'client/components/Button/Button';

test('With Jest snapshot, Button component renders as disabled', () => {
  const buttonComponent = renderer
    .create(<Button theme="homepageClick" disabled />)
    .toJSON();

  expect(buttonComponent).toMatchSnapshot();
});

test('With Jest snapshot, Button component renders as enabled', () => {
  const buttonComponent = renderer
    .create(<Button theme="homepageClick" disabled={false} />)
    .toJSON();

  expect(buttonComponent).toMatchSnapshot();
});
