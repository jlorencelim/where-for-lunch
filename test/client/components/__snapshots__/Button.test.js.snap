// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Jest snapshot, Button component renders as disabled 1`] = `
<button
  className="root homepageClick disabled"
  disabled={true}
/>
`;

exports[`With Jest snapshot, Button component renders as enabled 1`] = `
<button
  className="root homepageClick"
  disabled={false}
/>
`;
